lib/rspec/mocks/proxy.rb

```diff
@@ -228,6 +228,11 @@ def prepended_modules_of_singleton_class
         end
       end
 
+      # @private
+      def method_double_if_exists_for_message(message)
```

Comment:
Even just `method_double_if_exists_for(message)` - there's already a handful of `if` methods in Proxy, so that doesn't both me here too much, but `…for_message(message)` seems redundant